types/react-fontawesome/react-fontawesome-tests.tsx

```diff
@@ -14,3 +14,14 @@ class TestComponent extends React.Component {
     );
   }
 }
+
+class CSSTestComponent extends React.Component {
+  render() {
+    return (
+      <FontAwesome
+        name="rocket"
+        style={{backgroundColor: 'red', color: 'blue'}}
+      />
+    );
+  }
+}
```

Comment:
Hi vincaslt, Yes that seems to work well, so I've made the changes you suggested. Thanks!

I didn't get a failing test on the line ending however, as one exists:
![image](https://user-images.githubusercontent.com/8677029/29170761-09cefe42-7da8-11e7-8984-7ab1273da1a8.png)
I'm using Ubuntu, perhaps if you're on Windows the line endings are being checked out differently? Not sure ..